onnxruntime/test/optimizer/cse_test.cc

```diff
@@ -25,7 +25,7 @@ namespace {
 void ApplyCse(Model& model, unsigned num_steps = 1) {
   GraphTransformerManager graph_transformation_mgr(num_steps);
   ASSERT_TRUE(
-      graph_transformation_mgr.Register(onnxruntime::make_unique<CommonSubexpressionElimination>(), TransformerLevel::Level1).IsOK());
+      graph_transformation_mgr.Register(onnxruntime::make_unique<CommonSubexpressionElimination>( std::unordered_set<std::string>()), TransformerLevel::Level1).IsOK());
```

Comment:
You can skip the  changes in test files as you don't want to build the C/C++ tests.